evolver/util.py

```diff
@@ -14,16 +11,3 @@ def find_package_location(package=__project__):
 
 def find_repo_location(package=__project__):
     return Path(find_package_location(package) / os.pardir)
-
-
-def load_config_for_evolver(evolver: Evolver, configfile: Path):
-    """Loads the specified config file and applies to evolver."""
-    if configfile.exists():
```

Comment:
@amitschang My removing this so that this functionality is explicit, compounded with the mokeypatch that I added instead of your ``chdir`` caused the update app test to fail. This also then made me realize, as I had already brought to your attention, the scope of the testclient fixture was incorrect, changing that here such that the config loading and updating test was actually testing what we expected and not a previously saved config from another test, then required the evolver to be instantiated at runtime (app startup) and not import time. That then lead to the lifespan addition since order between instantiation and config loading could be better addressed, which then lead to adding it to the app, so that the instantiation from within a func was more elegant.